bin/git-missing

```diff
@@ -12,16 +12,28 @@ fi
 
 declare -a git_log_args=()
 declare -a branches=()
+declare pathspec=""
+declare parse_path=false
+
 for arg in "$@" ; do
 
+    if [[ $parse_path == true ]]; then
+        pathspec="$*"
```

Comment:
From this code, it looks like only a single `pathspec` is supported. If multiple are passed, it is treated as a single path. To fix this, `pathspec` should be an indexed array; then, you can do `pathspec+=("$@")` here.

Reply:
Excellent point! I was perhaps to focused on my usual use case of using just one path. I introduced your proposal in the just added commit. (b8488cd)